python/lsst/resources/davutils.py

```diff
@@ -2434,11 +2494,12 @@ def parse(self, body: bytes) -> list[DavProperty]:
         Parameters
         ----------
         body : `bytes`
-            XML-encoded response body to a PROPFIND request
+            XML-encoded response body to a PROPFIND request.
 
         Returns
         -------
-        responses : `List[DavProperty]`
+        responses : `list` [`DavProperty` ]
+            Parsed content of the response.
```

Comment:
This one will need to be fixed weather you decide to keep the spaces between the brackets and the item type or not. :wink: